config/initializers/blog.rb

```diff
@@ -0,0 +1,23 @@
+#See http://www.sitepoint.com/jekyll-rails/
+
+Rails.application.config.after_initialize do
+  Rails.logger = Logger.new(STDOUT)
+  begin
+    #make a spot for the site
+    dest = Rails.root.join('public/blog')
+
+    #generate the site
+    Jekyll::Site.new(
+    Jekyll.configuration({
```

Comment:
Redundant curly braces around a hash parameter.<br>Indent the first parameter one step more than the previous line.